holdings-api/src/main/java/com/okta/developer/holdingsapi/Holding.java

```diff
@@ -9,24 +9,27 @@ public String getCrypto() {
         return crypto;
     }
 
-    public void setCrypto(String crypto) {
+    public Holding setCrypto(String crypto) {
```

Comment:
@mraible not sure how you feel about this one.
I added it for simplicity, so the test code looks like:
```java
new Holding()
                    .setCrypto("crypto-1")
                    .setCurrency("currency-1")
                    .setAmount("amount-1")
```

but we could accomplish the same thing with an extra constructor or a few extra lines of code.

Reply:
I like fluid POJOs, so +1. You'll just have to make sure and explain the change in the post. ;)